frontend/src/pages/ForgotPassword/ForgotPassword.jsx

```diff
@@ -79,22 +142,46 @@ export function ForgotPassword() {
             ) : (
               <React.Fragment>
                 <h1 className={style["card-heading"]}>Trouble Logging In?</h1>
-                <form onSubmit={submitForgotPassword}>
+                <form
+                  onSubmit={() => {
+                    handleSubmit();
+                    submitForgotPassword();
+                  }}
+                >
                   <div className={style["inside-card"]}>
-                    <div className={style["forgot-password-input"]}>
-                      <input
-                        autocomplete="off"
-                        id="username"
-                        type="email"
-                        required="required"
-                        name="email"
-                        placeholder="Email"
-                        onChange={handleChange}
-                        className={style["input-forgot-password"]}
-                        onInvalid={(e)=>{ e.target.setCustomValidity("Enter a valid Email Id") }}
-                        onInput={(e)=>{ e.target.value?.match(new RegExp(/^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/)) && e.target.setCustomValidity("") }}
-                      />
-                      <i className="fas fa-at"></i>
+                    <div className={`form-group ${style["form-group"]}`}>
+                      <div className={style["forgot-password-input"]}>
+                        <input
+                          autocomplete="off"
+                          id="username"
+                          type="email"
+                          name="email"
+                          placeholder="Email"
+                          onChange={handleChange}
+                          required="required"
```

Comment:
Hey @AbhipsaGuru1012 as we don't need any default error message to be displayed anymore so I think we should remove it

Reply:
@Kajol-Kumari I am working on the latest changes. It is taking a bit of time.

Reply:
No worries, please take your time and keep us updated incase you need more than 3-5 days for this PR